AutoDefineAddon/autodefine.py

```diff
@@ -375,7 +377,7 @@ def _get_definition(editor,
                 to_print = re.sub(':', '', to_print)
                 # erase space between semicolon and previous word, if exists, and strip any extraneous whitespace
                 to_print = to_print.replace(" ; ", "; ").strip()
-                to_print += "<br>\n"
+                to_print += "<br>"
```

Comment:
I think we want to keep this newline for readability when looking at the raw html

Reply:
Well… I can understand the intent behind this `\n`, yet it produces unexpected behavior for a normal end user the way it is. Consider this test:

1. By using _Definition only_ button, insert just a definition into a field in your Anki note.
2. Select all inserted definition by _Ctrl+A_ and delete it.
3. Repeat step one to insert the definition into the field again.

You’ll see the inserted definition this time is preceded by a blank line. User has two choices to get rid of this blank line each time:
* They should manually delete this visible blank line after they erase and re-insert a definition
or
* Whenever they want to remove definition by selecting all text and erasing it, they should then press an extra _backspace_ to delete also the invisible `\n`.

Thus, the combination `<br>\n` does not work well. Nevertheless, I found another combination that resolves your referred issue as well as mine, and this is `\n<br>`.